adapters/adtrgtme/adtrgtme.go

```diff
@@ -32,9 +32,24 @@ func (v *adapter) MakeRequests(
 	[]*adapters.RequestData,
 	[]error,
 ) {
+	var ext openrtb_ext.ExtImpAdtrgtme
 	var requests []*adapters.RequestData
 	var errors []error
 
+	for i, imp := range openRTBRequest.Imp {
+		var err error
+		ext, err = v.getImpressionExt(&imp)
+		if err != nil {
+			errors = append(errors, err)
+			break
```

Comment:
Do you intend to break out of the imp loop if `getImpressionExt` fails or would you like to continue for remaining imps?

Reply:
I don't understand what you mean.

Reply:
So the `break` statement here breaks out of the for loop. This would mean that if there are remaining `openRTBRequest.Imp[i]` impression then it will not be processed. Do you intend to keep this behaviour or would you like to continue to process for remaining impressions?

Reply:
```
if err != nil {
			errors = append(errors, err)
			continue
}
```
The above code will continue to process for the remaining impressions in the for loop.

Reply:
I answered below https://github.com/prebid/prebid-server/pull/2637#discussion_r1145785244

Reply:
This comment is different from https://github.com/prebid/prebid-server/pull/2637#discussion_r1145785244
Rephrasing - Suppose your adapter has request for multiple impressions in single request - imp1, imp2, imp3. 
In the for loop, getImpressionExt returns an error then line 44 `break` statement will not process imp2 and imp3 impressions. Would that be ok for your adapter since it may be the case where imp2 and imp3 were the valid Imps and your adapter could have bid on it.
Also could you please add a test case for multiple impressions?